test/integration/derivedcr.go

```diff
@@ -162,6 +162,14 @@ func newDerivedCR(
 			assert.Equal(t, catalogEntry.Status.TenantCRD.Kind, dcr.Status.DerivedCR.Kind)
 			assert.Equal(t, catalogEntry.Status.TenantCRD.Plural, dcr.Status.DerivedCR.Plural)
 		}
+
+		// Check the Elevator dynamic webhook service is deployed.
+		webhookService := &corev1.Service{}
```

Comment:
This is a bit too specific for this test. Instead, you should test whether the webhook works properly by issuing a dry-run call/trying to create derivedCRD instance.

Reply:
yeah, we also need to do the same thing for the catapult PR, since it will also involve e2e operator, so I just want to have a really specific one to check if the webhook service is created, and we can add e2e tests for that later.

Reply:
ok, will deal with it later